example/src/Examples/WithBottomSheet.tsx

```diff
@@ -0,0 +1,57 @@
+import BottomSheet, { BottomSheetFlatList } from '@gorhom/bottom-sheet';
```

Comment:
nice ! can we also add it to the doc website? with explanations code snippet and maybe a gif?

Reply:
added